src/adapters/loading.py

```diff
@@ -368,6 +369,19 @@ def _rename_legacy_weights(self, k):
             k = k.replace(old, new)
         return k
 
+    def _fix_backward_compat(self, config):
+        # Fix error in previous versions for LoRA/ (IA)^3
+        if config.get("version", "").startswith("adapters.") and Version(config["version"][9:]) < Version("1.1.0"):
```

Comment:
```suggestion
        ADAPTER_PREFIX = "adapters."
        MIN_VERSION = Version("1.1.0")

        version = config.get("version", "")
        if version.startswith(ADAPTER_PREFIX) and Version(version[len(ADAPTER_PREFIX):]) < MIN_VERSION:
```

Reply:
Would remove unnecessary double lookup in config and has no hardcoded version suffix